setup.py

```diff
@@ -18,7 +18,7 @@
 from setuptools import setup, find_packages
 import sys
 
-install_requires = ['intelhex', 'six', 'enum34']
+install_requires = ['intelhex', 'six', 'enum34', 'future', 'websocket-client']
```

Comment:
Was the 'six' module insufficient for backwards compatibility?  Or was future easier to use / cleaner?

Also, adding websocket-client will force all the users of pyOCD to install it.  Maybe its small enough it doesn't matter.